### DISCUSSION TITLE
libtiff not found

### DISCUSSION THREAD

**Discussion by haukemac** [ImageMagick/ImageMagick#7316] [Help]
### ImageMagick version

7.1.1.-32

### Operating system

Linux

### Operating system, version and so on

centos7

### Description

Hello,
when compiling ImageMagick with own libtiff and xz library in a custom path, during the configuration lifbtiff is not found.
Installing xz-devel from repository fixes that.

### Steps to Reproduce

Build with own libtiff and xz library

**Comment by urban-warrior** [comment]
Check `config.log` and search for `libtiff`.   The log reveals why the TIFF delegate library validation failed.  Fix the issue and try `./configure` once more.   Hopefully the delegate library will now validate and you can continue with your build / install.  You should now be able to read and write TIFF images.
